app/models/claim/geckoboard_event.rb

```diff
@@ -2,32 +2,34 @@
 
 class Claim
   class GeckoboardEvent
-    attr_reader :claim, :event_type, :performed_at
+    attr_reader :claims, :event_type, :date_field
```

Comment:
I had to dig quite a bit to find out what date_field is expected to be, I wonder if a different name could help, something like `event_timestamp`?

Reply:
Like it 👍